src/App.tsx

```diff
@@ -4,44 +4,50 @@ import { SubmitLink } from "./components/SubmitLink";
 import { Message } from "./components/Message";
 import { Listing } from "./components/Listing";
 import { ListItem } from "./types/ListItem";
+import "./App.css";
 
-const App = (): JSX.Element => {
+const App: React.FC<Record<string, never>> = () => {
```

Comment:
Using `Record` in generic parameter seems a bit suspicious.
When trying to dig deeper, it seems like `Rect.FC<...>` seems to be [discouraged](https://github.com/typescript-cheatsheets/react#function-components) by some wise people. While it's not a big deal imo. Why was not the `(): JSX.Element` enough?

Reply:
I didn't know of the React.FC downsides, thanks for that!
The solution I chose was temporary detour from errors accompanied with `(): JSX.Element` not needing import of `React` ([more here](https://github.com/microsoft/TypeScript/issues/41882)), however all seems functioning again with `(): JSX.Element` after I reinstalled node modules.